README.md

```diff
@@ -71,7 +77,9 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+
+This would make  more sense as a model method, because controllers aren't supposed to be filled with this type of logic/heavy lifting. Also, the model is where the classes the app is working with are defined and this ranking system would fall in line with that reasoning.
```

Comment:
Actually, we were looking for a *helper* method. Helpers are intended to render HTML, like `form_for` and `link_to`, or render other things that are important only in a view. Model methods are more concerned with doing things to the model itself and how its data is accessed.

You argued your point well, though, so I'll give you a :white_check_mark: .